aerleon/lib/nacaddr.py

```diff
@@ -238,23 +259,23 @@ def AddComment(self, comment=''):
 IPType = Union[IPv4, IPv6]
 
 
-def _InNetList(adders, ip):
+def _InNetList(adders: List[IPv4], ip: IPv4) -> bool:
     """Returns True if ip is contained in adders."""
     for addr in adders:
         if ip.subnet_of(addr):
             return True
     return False
 
 
-def IsSuperNet(supernets, subnets):
+def IsSuperNet(supernets: List[IPv4], subnets: List[IPv4]) -> bool:
```

Comment:
Is this code actually restricted to IPv4 only? IT just calls `_InNetList` which calls `ip.subnet_of()`, which is implemented on both kinds of IP networks. Better to scope the type restriction to `_BaseNetwork`?

Reply:
Or `Union[IPv4, IPv6]`

Reply:
Ahh good catch, I didn't look very closely it seems.